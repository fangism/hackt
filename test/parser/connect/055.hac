// "055.hac" should fail: size mismatch

defproc portaproc(bool b, c[4]; int i, j[2]);

bool a;
bool x[4];
int q;
int r[2];

portaproc p, s;

p(x[2], x, x, r);		// 3rd argument type mismatch
s( ,  ,  ,  );

