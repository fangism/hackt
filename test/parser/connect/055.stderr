Types do not match! got: int<32> and: bool<>.
ERROR: actual instance reference 3 of port connection doesn't match the formal type/size.
	got: bool<>[4], expected: int<32>
At least one error in port connection.  
HALT: at least one error in port connection list.  [12:1..16]
